topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.md

```diff
@@ -0,0 +1,201 @@
+
+In this tutorial, we will use a set of tools from the **Genetic Design - BASIC Assembly Workflow** (https://galaxy-synbiocad.org) which will enable you to design plasmids implementing metabolic pathways for the bioproduction of lycopene in _E.coli_ using the [BASIC assembly method](https://doi.org/10.1021/sb500356d), and to generate scripts enabling the automatic build of these plasmids using an Opentrons liquid handler robot. After downloading these scripts onto a computer connected to an Opentrons, one can perform the automated construction of the plasmids at the bench.
+
+The workflow scheme we will use is shown below. First, we will run the steps of this workflow individually so as not to neglect the understanding of the intermediate steps as well. Then, we will run the workflow automatically so that it itself retrieves the outputs from the previous step and gives them as input to the next tool.
+
+![Genetic Design - BASIC Assembly Workflow](../../images/basic_assembly_workflow.png)
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Data Preparation
+
+First we need to upload and prepare the following inputs to analyze:
+
+- One SBML file modeling a heterologous pathway producing lycopene such as those produced by the Pathway Analysis Workflow (https://galaxy-synbiocad.org).
+
+- "parts_for_lycopene.csv" file listing the parts to be used (linkers, backbone and promoters) in the constructions.
+
+- Two YAML files providing two examples of settings, i.e. providing the identifiers of the laboratory equipment and the parameters to be used in the Opentrons scripts.
+
+## Get data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial named *Genetic Design - BASIC Assembly Analysis*.
+> 2. Import the input files from [Zenodo]({{ page.zenodo_link }}):
+>
+>    ```
+>    https://zenodo.org/api/files/f58fe126-0ca2-4ca5-bc4b-0063f3a8951c/dnabot_london_settings.yaml
+>    https://zenodo.org/api/files/f58fe126-0ca2-4ca5-bc4b-0063f3a8951c/dnabot_paris_settings.yaml
+>    https://zenodo.org/api/files/f58fe126-0ca2-4ca5-bc4b-0063f3a8951c/parts_for_lycopene.csv
+>    https://zenodo.org/api/files/f58fe126-0ca2-4ca5-bc4b-0063f3a8951c/rp_002_0011.xml
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Rename the datasets if you want to shorten the file names.
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+{: .hands_on}
+
+# Find enzymes using **Selenzyme**
+
+At first, a pathway generated by the Pathway Analysis workflows (https://galaxy-synbiocad.org) is provided as an SBML file to the [Selenzyme](https://doi.org/10.1093/bioinformatics/bty065) tool. Selenzyme searches for enzymes corresponding to each reaction of the pathway. It performs a reaction similarity search in the reference reaction database [MetaNetX](https://www.metanetx.org) and outputs an updated SBML file annotated with the enzyme UniProt IDs.
```

Comment:
you might use a `{% cite .. %}` here as well

Reply:
Ok, done.